superflore/docker.py

```diff
@@ -20,9 +20,10 @@
 
 
 class Docker(object):
-    def __init__(self, dockerfile, name):
+    def __init__(self, dockerfile=None, name=None):
```

Comment:
I don't see `self.name` being used anywhere is it necessary here?

Reply:
I guess I missed that. It certainly isn't used.